project_closing/models/analytic.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013 Camptocamp Author Vincent Renaville
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, api
```

Comment:
Actually it is (pay attention to the comment in the code-block here:https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports)
I think it is nearly never enforced: I tried in the past and all developers complaining that it was not necessary since there is tool for it (isort).